data/json/items/comestibles/vegetable.json

```diff
@@ -0,0 +1,312 @@
+[
+  {
+    "id": "broccoli",
+    "copy-from": "vegetable_leafy",
+    "type": "COMESTIBLE",
+    "name": "broccoli",
+    "name_plural": "broccoli",
+    "description": "It's a bit tough, but quite delicious.",
+    "weight": 170,
+    "volume": 1,
+    "price": 400,
+    "spoils_in": 160,
+    "nutrition": 30,
+    "healthy": 2,
+    "flags": [ "EATEN_HOT" ]
```

Comment:
Was this in before? Heating up a broccoli over a hotplate isn't exactly the proper way to cook it